apps/extension/src/core/llm/model.ts

```diff
@@ -44,6 +44,8 @@ export interface RequestOptions {
   timeout?: number
   user_identifier?: string | null
   max_tokens?: number | null
+  prompt_cost?: number | null
```

Comment:
these two attrs seem like they're innate to the model and not the request, so I think they should go in `ModelConfig`. (RequestOptions are the properties that are configurable on a per-request basis)

One weird bit is the fact that they're dependent on `model`. One way of dealing with this is by adding a function to ModelConfig that computes the total cost based on the request, similar to `getPath: (request: RequestData) => string`

Reply:
That makes sense. I'll implement that change and make it work like getPath.

Reply:
I moved the two attributes to the model. I currently have a function that computes the total cost based on the request in TransactionManager, because we also need to compute the number of tokens in the prompt to get the total cost. Can you take a look and let me know if I should change it?

Reply:
I've attempted to implement this based on what you suggested. I have a question for you about this, which I've written here: https://github.com/ThePatShea/window.ai/pull/1/files#r1182151955